tests/node/test_wakunode_filter.nim

```diff
@@ -0,0 +1,116 @@
+{.used.}
+
+import
+  std/sequtils,
+  std/options,
+  stew/shims/net as stewNet,
+  testutils/unittests,
+  chronicles,
+  chronos,
+  libp2p/crypto/crypto
+
+import
+  std/[options, tables, sequtils],
+  testutils/unittests,
+  chronos,
+  chronicles,
+  os,
+  libp2p/peerstore
```

Comment:
Is there a reason for having two separate import statements and with some repeated modules?
I think it's better to combine them into one import statement without duplicates

Reply:
I followed the style I saw in sibling files. I think the reasoning is to separate `builtin` and `third-party` imports from local ones.

Reply:
Yes, but in this case there's 3 import statements instead of 2 and `chronos`, `chronicles` and some other modules are being imported twice:

https://github.com/waku-org/nwaku/blob/b354235eba77e29911f7a6e641932cba3eeea61c/tests/node/test_wakunode_filter.nim#L3-L31

Reply:
Oh, that. It seems my brain just read the first half and assummed the second part. You're completely right, changing that.